accounts/admin.py

```diff
@@ -29,18 +31,55 @@
 from django.http import HttpResponseRedirect
 from django.core.urlresolvers import reverse
 from django.contrib import messages
+from django.conf import settings
 
 
 def disable_active_user(modeladmin, request, queryset):
-    for user in queryset:
-        user.profile.delete_user(remove_sounds=True)
+    if request.POST.get('confirmation', False):
+        gm_client = gearman.GearmanClient(settings.GEARMAN_JOB_SERVERS)
+        for user in queryset:
+            gm_client.submit_job("delete_user",
+                    json.dumps({'user_id':user.id, 'action': "delete_user_delete_sounds"}),
+                wait_until_complete=False, background=True)
+        messages.add_message(request, messages.INFO,
+             '%d Soft deleted users including her sounds' % (queryset.count()))
```

Comment:
The message should talk about the future? "users will be soft deleted asynchronously".
The same applies to the other messages of this kind.